### PR TITLE
bind-tools and remaining dependencies

### PR DESCRIPTION
**Pull request by rawlingsj**
This seems to be the last of the packages needed for bind + subpackages such as bind-tools.

Packages include

- [x] bind
- [x] fstrm
- [x] protobuf-c
- [x] protobuf
- [x] dns-root-hints
- [x] dnssec-root
- [x] gnupg
- [x] gnutls
- [x] nettle
- [x] npth
- [x] libassuan
- [x] libksba
- [x] libidn2
- [x] libkcapi

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -352,6 +352,20 @@ $(eval $(call build-package,at-spi2-core,2.46.0-r0))
...
+$(eval $(call build-package,libkcapi,1.4.0-r0))
+$(eval $(call build-package,libidn2,2.3.4-r0))
+$(eval $(call build-package,libksba,1.6.3-r0))
+$(eval $(call build-package,libassuan,2.5.5-r0))
+$(eval $(call build-package,npth,1.6-r0))
+$(eval $(call build-package,nettle,3.8.1-r0))
+$(eval $(call build-package,gnutls,3.7.8-r0))
+$(eval $(call build-package,gnupg,2.2.41-r0))
+$(eval $(call build-package,dnssec-root,20190225-r0))
+$(eval $(call build-package,dns-root-hints,2022062901-r0))
```
Comment (rawlingsj) [inline Makefile:364]: does this make sense?

File: dns-root-hints.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - gpg
+      - curl
+      - gpgv
+pipeline:
+  - runs: |
+      # Convert the key to the GPG keyring that can be used with gpgv.
+      gpg --dearmor -o verisign-grs-nstld-key.gpg verisign-grs-nstld-key.asc
```
Comment (rawlingsj) [inline dns-root-hints.yaml:25]: this seemed a little strange but alpine seems to take the same approach

File: dns-root-hints/verisign-grs-nstld-key.asc [hunk 3]
```diff
@@ -0,0 +1,50 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```
Comment (rawlingsj) [inline dns-root-hints/verisign-grs-nstld-key.asc:1]: This is also the same approach as Alpine https://git.alpinelinux.org/aports/plain/main/dns-root-hints/verisign-grs-nstld-key.asc

File: dnssec-root.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  - uses: fetch
+    with:
+      expected-sha256: 6c704b7b3f13e828d87de8b971b99721e46586fd3a8a547cf4e7b748cbe3d493
+      uri: https://github.com/iana-org/get-trust-anchor/archive/67c11662510f5e2db6e6517228e80b794950c43f.tar.gz
+  - runs: |
+      python3 get_trust_anchor.py
+      sort -k 6 ksk-as-dnskey.txt* > trusted-key.key
+
+      # abump on alpine updates the version?  https://wiki.alpinelinux.org/wiki/Include:Abump
+      # abump dnssec-root-$(date "+%Y%m%d")
```
Comment (rawlingsj) [inline dnssec-root.yaml:32]: Can we just remove these?  Wasn't totally sure if we need this.

File: gnupg.yaml [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+      - libtool
+      - libusb-dev
+      - npth-dev
+      - openldap-dev
+      - sqlite-dev
+      - texinfo
+      - zlib-dev
+      - gnutls-dev
+      - glibc-iconv
+      # - pinentry TODO skipping for now as seems to require gtk+3.0, gcr, libproxy and mesa
```
Comment (rawlingsj) [inline gnupg.yaml:42]: I skipped this dependency as it was starting to open a can of worms.  Everything's compiling without it, so maybe we add later when needed?

File: dns-root-hints/update-dns-root-hints [hunk 6]
```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+# originated from https://git.alpinelinux.org/aports/plain/main/dns-root-hints/update-dns-root-hints
```
Comment (rawlingsj) [inline dns-root-hints/update-dns-root-hints:2]: is this needed?

**Review by jdolitsky — APPROVED** [review]
wow

File: dns-root-hints/update-dns-root-hints (hunk 6 above)
Comment (kaniini) [reply dns-root-hints/update-dns-root-hints:2]: Probably not, but harmless to keep.

**Review by kaniini — APPROVED** [review]

**Pull request merged**